### PR TITLE
fix direction for Arabic language

### LINKED ISSUE #126: translation: language changed and fix direction
- [x] fix language change

- [x] change the direction when Arabic has been chosen

### PR DESCRIPTION
**Pull request by Jamil-Bailony**
- [x]  fix language change

- [x]  change the direction when Arabic has been chose
 

Closes #126

### CODE REVIEW

File: src/i18n.config.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+  document.body.setAttribute("dir", direction);
+
+  document.getElementById("root").style.textAlign =
+    direction === "rtl" ? "right" : "left";
+}
+
 i18n.on("languageChanged", function (lng) {
   localStorage.setItem("lng", lng);
+  if (lng === "ar") changeDirection("rtl");
+  else changeDirection("ltr");
```
Comment (louisrli) [inline src/i18n.config.js]: i have written about this before

Comment (louisrli) [reply src/i18n.config.js]: pls fix

Comment (louisrli) [reply src/i18n.config.js]: braces

File: src/i18n.config.js [hunk 2]
```diff
@@ -1,11 +1,19 @@
 import i18n from "i18next";
 import { initReactI18next } from "react-i18next";
 import LanguageDetector from "i18next-browser-languagedetector";
-
 import Backend from "i18next-http-backend";
 
+function changeDirection(direction) {
```
Comment (louisrli) [inline src/i18n.config.js]: setdirection

**Review by louisrli — APPROVED** [review]
if your AR direction wasn't working before i get the feeling this is gonna break a lot of things on the site but oh well

**Review by hiba-machfej — APPROVED** [review]

**Comment by hiba-machfej** [comment]
> if your AR direction wasn't working before i get the feeling this is gonna break a lot of things on the site but oh well

I am working on the issues right now.

**Comment by louisrli** [comment]
that's definitely one of those things that should have been implemented
earlier, not after code freeze :) let's get that in by end of today please,
work with your teammates if theres a lot. if you can't finish it by today
or its huge changes i'm tempted to say just leave broken directionality in

**Pull request merged**